hw_5/pandas_script.py

```diff
@@ -0,0 +1,267 @@
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import re
+import seaborn as sns
+from matplotlib import patches
+import matplotlib.font_manager as font_manager
+
+# FIRST TASK
+
+
+# I. Function for gff reading
+def read_gff(input_gff):
+    gff_dataframe = pd.read_csv(input_gff, sep='\t', header=1,
+                                names=['chromosome', 'source', 'type', 'start',
+                                       'end', 'score', 'strand', 'phase', 'attributes'])
```

Comment:
Твой код учитывает только случай, где комментарий занимает только первую строку. Однако чаще всего в файлах данного формата могут быть комментарии на ммного строк, их нужно учитывать. Благо в пандасе это делается супер просто
```suggestion
    gff_dataframe = pd.read_csv(input_gff, sep='\t', comment="#",
                                names=['chromosome', 'source', 'type', 'start',
                                       'end', 'score', 'strand', 'phase', 'attributes'])
```